docs/whats-new/production-release-notes.md

```diff
@@ -2,6 +2,61 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Production release 22 March 2023
+
+Frontend version: 1.0.0-20230315.0325 | Backend version: 1.54.0-230309.0902
```

Comment:
we released a hotfix yesterday, the latest BE version for prod is 1.54.3-230321.0659

Reply:
so i should change only in the prod release note, right? that is done.